forc-plugins/forc-doc/src/render.rs

```diff
@@ -161,19 +161,35 @@ fn page_from(rendered_content: Box<dyn RenderBox>) -> String {
 }
 
 /// Basic HTML header component
-fn html_head(location: String, decl_ty: String, decl_name: String) -> Box<dyn RenderBox> {
+fn html_head(
+    module_depth: usize,
+    location: String,
+    decl_ty: String,
+    decl_name: String,
+) -> Box<dyn RenderBox> {
+    let prefix = module_depth_to_path_prefix(module_depth);
+    let mut normalize = prefix.clone();
+    let mut swaydoc = prefix.clone();
+    let mut ayu = prefix;
+    normalize.push_str("normalize.css");
+    swaydoc.push_str("swaydoc.css");
+    ayu.push_str("ayu.css");
```

Comment:
Not sure if this looks any better to you but I recently realized `String` implements `Add` trait so you can do something like:

```suggestion
    let normalize = prefix.clone() + "normalize.css";
    let swaydoc = prefix.clone() + "swaydoc.css";
    let ayu = prefix + "ayu.css";
```

Reply:
anything resembling javascript just sends me back to vietnam tbh

Reply:
https://tenor.com/view/vietnam-flashback-cat-gif-18770750